langchain_mcp_adapters/resources.py

```diff
@@ -0,0 +1,93 @@
+import base64
+
+from langchain_core.documents import Document
+from langchain_core.documents.base import Blob as LCBlob
+from mcp import ClientSession
+from mcp.types import BlobResourceContents, ResourceContents, TextResourceContents
+
+
+def convert_mcp_resource_to_langchain_document(
+    resource_uri: str,
+    contents: ResourceContents,
+) -> Document | LCBlob:
```

Comment:
let's just return a list of `Blob`, no need for a mixed type here, same for the other functions